docs/content/en/docs/configuration/keys.md

```diff
@@ -326,6 +326,8 @@ The table below describes all supported configuration keys.
 | [`cors-expose-headers`](#cors)                       | headers                                 | Path    |                    |
 | [`cors-max-age`](#cors)                              | time (seconds)                          | Path    |                    |
 | [`cpu-map`](#cpu-map)                                | haproxy CPU Map format                  | Global  |                    |
+| [`default-backend-redirect`](#default-redirect)     | Default Location to Redirect a user if the resource does not exists     | Global  |                    |
+| [`default-backend-redirect-code`](#default-redirect)         | Response Code to be used when redirecting a user to the default location | Global  | `302`                   |
```

Comment:
Realign the table, adding a space in the former and removing some of them in the later. Description can also be as short as `Location` and `HTTP status code` or something like this.